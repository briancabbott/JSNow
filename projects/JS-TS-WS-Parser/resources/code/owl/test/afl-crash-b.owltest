---a =
 b | identifier : b
.operators prefix
 identifier : c
b = J\:p
J = '' : p
