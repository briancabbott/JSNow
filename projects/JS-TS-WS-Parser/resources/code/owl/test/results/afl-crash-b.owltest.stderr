error: expected more text after the last token


